container/BUILD

```diff
@@ -150,6 +148,8 @@ TEST_DATA = [
     for t in TEST_TARGETS
 ] + [
     "//testdata:stamped_bundle_test",
+    "//testdata/import:pause_piecemeal.tar",
```

Comment:
why do these needed to be moved from TEST_TARGETS to TEST_DATA (and they just be left as targets)?

Reply:
because they have different paths then the rest

Reply:
I'm not sure they need to be here just because they have different paths. Did you try to leave them where they were with the new path? (if you did and got an error, could you let me know what kind of error it was, it does not make sense to have to have these be test data dependencies as opposed to test targets.

Reply:
I'm not sure they need to be here just because they have different paths. This struct is declaring data deps of the tests, the one above is declaring targets. If you add the dep here, any changes to this test would cause rerunning all other tests (as they depend on this one) if you add the dep above, changes to this test would only cause rerunning this test.